java/testing.bzl

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
Do we have a way of keeping .bzl files private through enforcement? If not, can we more clearly mark this as internal-only in the file or directory name?

Reply:
I thought they were but they aren't https://docs.bazel.build/versions/master/visibility.html#visibility-of-bzl-files I'll move to an "internal" folder.